recipes/spirv-headers/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.198.0":
+    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.2.198.0.tar.gz"
+    sha256: "3301a23aca0434336a643e433dcacacdd60000ab3dd35dc0078a297c06124a12"
   "cci.20211010":
     url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/814e728b30ddd0f4509233099a3ad96fd4318c07.tar.gz"
```

Comment:
this version (with a typo, because it should have been `cci.20211110`) should be removed, it's the commit of 1.2.198.0